go.mod

```
@@ -1,5 +1,10 @@
-module github.com/milligan22963/passphrase
+module github.com/asphaltbuffet/passphrase
```

this needs to stay as your repo
```suggestion
module github.com/milligan22963/passphrase
```